go/libkb/leveldb_test.go

```diff
@@ -229,20 +249,16 @@ func TestLevelDb(t *testing.T) {
 					defer wg.Done()
 
 					tr, err := db.OpenTransaction()
-					if err != nil {
-						fmt.Println(err)
-						t.Fatal(err)
-					}
+					require.NoError(t, err)
```

Comment:
the existing test is also broken, but not supposed to call t.Fatal inside a goroutine.  (and `require` package uses `t.FailNow` which is same)...   `if err != nil { t.Error(err) }` would be fine.

Reply:
fixed in https://github.com/keybase/client/pull/16263